psl/psl-core/src/validate/validation_pipeline/validations/relation_indexes.rs

```diff
@@ -0,0 +1,52 @@
+use diagnostics::DatamodelWarning;
+use parser_database::ast::WithSpan;
+use parser_database::{
+    ast::ModelId,
+    walkers::{RelationFieldWalker, Walker},
+};
+
+use crate::{datamodel_connector::RelationMode, validate::validation_pipeline::context::Context};
+
+pub(super) fn validate_missing_relation_indexes(
+    model: Walker<'_, ModelId>,
```

Comment:
Also missed that previously, but this line is not necessary. You can get to the model through `relation_field.model()`.

Reply:
Thanks